frontends/web/package.json

```diff
@@ -41,6 +41,6 @@
     "sortablejs": "^1.10.1"
   },
   "browserslist": [
-    "last 2 Chrome versions"
+    "chrome >= 65"
```

Comment:
The last 2 versions of chrome today I believe are 85 and 86 so it makes sense that our poor qt webengine with features supported only at v65 possibly fails with some syntax errors for js code that targets v85 or v86.